modules/ivc_champva/app/models/ivc_champva/vha_10_7959a.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
+  class VHA107959A
+    include Virtus.model(nullify_blank: true)
+
+    attribute :data
+
+    def initialize(data)
+      @data = data
+    end
+
+    def metadata
+      {
+        'veteranFirstName' => @data.dig('veteran', 'full_name', 'first'),
+        'veteranLastName' => @data.dig('veteran', 'full_name', 'last'),
+        'fileNumber' => @data.dig('veteran', 'va_file_number').presence || @data.dig('veteran', 'ssn'),
+        'zipCode' => @data.dig('veteran', 'address', 'postal_code'),
+        'source' => 'VA Platform Digital Forms',
+        'docType' => @data['form_number'],
+        'businessLine' => 'CMP'
```

Comment:
Need to add `'primaryContactInfo' => @data['primary_contact_info']`

Reply:
I think there are also some other ones missing refer to other models as a reference